<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>libroiva</class>
 <widget class="QDialog" name="libroiva" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Libro de IVA SOPORTADO</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&lt;b>EJERCICIO:&lt;/b></string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ejerciciocomboBox" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="consultarpushButton" >
       <property name="text" >
        <string>&amp;Consultar</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-36.png</normaloff>:/images/librok-36.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimirpushButton" >
       <property name="text" >
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/graf/Print.png</normaloff>:/graf/Print.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="icon" >
        <iconset resource="libroK.qrc" >
         <normaloff>:/images/librok-52.png</normaloff>:/images/librok-52.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="libroK.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>libroiva</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
